### PR TITLE
v1:B-42833,junits

### PR DESCRIPTION
**Pull request by rahulstar7**

### CODE REVIEW

File: src/main/java/com/buildmaven/add/Test.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.buildmaven.add;
+
+public class Test {
+
+	public static void main(String[] args) {
+
```
Comment (BharatMarkad) [inline src/main/java/com/buildmaven/add/Test.java:6]: Please use standard comments for method & class level @rahulstar7 
As per discussion with @gdpatil39  plz resove this issue

Comment (gdpatil39) [inline src/main/java/com/buildmaven/add/Test.java:6]: please format the code as per standers

Comment (gdpatil39) [reply src/main/java/com/buildmaven/add/Test.java:6]: please format the code as per standers

File: src/main/java/com/buildmaven/add/Test.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package com.buildmaven.add;
+
+public class Test {
+
```
Comment (BharatMarkad) [inline src/main/java/com/buildmaven/add/Test.java:4]: Plz check the class name @rahulstar7

File: src/main/java/com/buildmaven/add/Test.java (hunk 1 above)
Comment (rahulstar7) [reply src/main/java/com/buildmaven/add/Test.java:6]: done

Comment (rahulstar7) [reply src/main/java/com/buildmaven/add/Test.java:6]: done

**Pull request merged**